src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -111,12 +121,10 @@ public void start(Map<String, String> properties) {
       offsets = context.offsetStorageReader().offsets(partitions);
     }
 
-    String schemaPattern
-        = config.getString(JdbcSourceTaskConfig.SCHEMA_PATTERN_CONFIG);
```

Comment:
The schema pattern and some of the other config properties are no longer needed by the task. However, they may be needed by the dialect, so if they're needed that's where they're used.